server/http/controller.go

```diff
@@ -773,8 +773,10 @@ func (c *controller) UpdateTestRun(ctx context.Context, testID string, runID int
 	existingRun.TriggerResult = run.TriggerResult
 	existingRun.Trace = traces.MergeTraces(existingRun.Trace, run.Trace)
 	existingRun.State = run.State
+	existingRun.LastError = errors.New(testRun.LastErrorState)
 
 	if executor.RunWasUserCancelled(existingRun) {
+		existingRun.LastError = nil
```

Comment:
So even if something went wrong, we gonna ignore it if the process was canceled, right?

Reply:
If there was an error, it would happen before the cancellation, so the test would be already stopped and the cancelation would be ignored before this